python/activity_stream/dialog_reply.py

```diff
@@ -32,14 +32,44 @@ def __init__(self, parent, bg_task_manager, note_id, allow_screenshots=True):
         # now load in the UI that was created in the UI designer
         self.ui = Ui_ReplyDialog() 
         self.ui.setupUi(self)
+
+        self._note_id = note_id
         
         self.ui.note_widget.set_bg_task_manager(bg_task_manager)        
         self.ui.note_widget.data_updated.connect(self.close_after_create)
         self.ui.note_widget.close_clicked.connect(self.close_after_cancel)
         self.ui.note_widget.set_current_entity("Note", note_id)
         self.ui.note_widget.allow_screenshots(allow_screenshots)
         
-        self.setWindowFlags(QtCore.Qt.Dialog | QtCore.Qt.FramelessWindowHint)        
+        self.setWindowFlags(QtCore.Qt.Dialog | QtCore.Qt.FramelessWindowHint)
+
+    @property
+    def note_widget(self):
+        """
+        Returns the underlying note_input_widget.NoteInputWidget.
+        """
+        return self.ui.note_widget
+
+    def _get_note_id(self):
+        """
+        Gets the entity id number for the parent Note entity
+        being replied to.
+
+        :returns:   int
+        """
+        return self._note_id
+
+    def _set_note_id(self, note_id):
+        """
+        Sets the entity id number for the parent Note entity
```

Comment:
Sorry, doh, meant to add this comment to the `_get_note_id` method, not the `_set_note_id` method.